### ISSUE TITLE
Down chevron and color picker button icons have insufficient contrast against their background on mouse hover

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#875]
The following AI-windows mode(s) have this issue:
- Dark mode
- High Contrast White mode
- High Contrast Black mode

**Screenshots or .GIF**
#### Screenshots of the issues in High Contrast White mode
![22 1 1](https://user-images.githubusercontent.com/22068434/88357059-cb323680-cd1e-11ea-959b-ffbad834a805.png)

![22 1 2](https://user-images.githubusercontent.com/22068434/88357065-ce2d2700-cd1e-11ea-8deb-bf02ba280088.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by DaveTryon** [comment]
The mouse hover of the eyedropper was fixed with #970. Looking at the color picker dropdown now

**Comment by DaveTryon** [comment]
Pushing this back onto the pile

**Comment by DaveTryon** [comment]
@iamrafan, this is fixed in the release available at https://github.com/microsoft/accessibility-insights-windows/releases/tag/v1.1.1438.01. Please validate and close at your convenience. Thanks!

**Comment by iamrafan** [comment]
Validated the fix in Canary Release: v1.1.1443.01. All 6 modes are now compliant
